api_app/core/models.py

```diff
@@ -5,25 +5,17 @@
 from django.utils import timezone
 from django.contrib.postgres import fields as pg_fields
 
-from enum import Enum
-
-
-class Statuses(Enum):
-    FAILED = 0
-    PENDING = 1
-    RUNNING = 2
-    SUCCESS = 3
-    KILLED = 4
-
 
 class AbstractReport(models.Model):
-    # constants
-    Statuses = Statuses
+    class Status(models.TextChoices):
```

Comment:
define it outside of `AbstractReport` but add only a reference to it inside the class.